torch/csrc/Module.cpp

```diff
@@ -450,6 +450,12 @@ PyObject *THPModule_setBenchmarkCuDNN(PyObject *_unused, PyObject *arg)
 {
   THPUtils_assert(PyBool_Check(arg), "set_benchmark_cudnn expects a bool, "
           "but got %s", THPUtils_typename(arg));
+#ifdef USE_ROCM
```

Comment:
Actually we use `defined(__HIP_PLATFORM_HCC__)` in C++ to detect ROCm builds. `USE_ROCM` is used in cmake.

Reply:
Updated define.